linchpin/provision/roles/aws/.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: python
+dist: xenial
+python:
+  - "2.7"
+  - "3.7"
+cache: pip
+services:
+  - docker
+
+install:
+  - pip install -U pip
+  - pip --version
+  - pip install ansible ansible-lint yamllint flake8 molecule docker
+  - ansible --version
+  - ansible-lint --version
+  - yamllint --version
+  - flake8 --version
+  - molecule --version
+
+script:
+  # every scenario with a converge playbook should be linted
+  - molecule lint -s openstack
```

Comment:
aws?

Reply:
I think this means "test this role from an openstack host".  It doesn't really matter for us yet since we're not using the molecule tests yet.  I can delete it though since we'll probably only use the docker/podman provider for molecule anyway

Reply:
I think it runs linting against "openstack" folder?